src/core/server.ts

```diff
@@ -9,7 +9,7 @@ import { GraphQLSchema } from 'graphql';
 import { Binding } from 'graphql-binding';
 import { Server as HttpServer } from 'http';
 import { Server as HttpsServer } from 'https';
-import opn = require('opn');
+const opn = require('opn');
```

Comment:
Does this syntax not work on Windows? If so, we should probably find a point rule to prevent it

Reply:
Yes, it gives error `error TS7016: Could not find a declaration file for module 'opn'...`.
I think a better way will be to update `opn` to `open` because [`opn` is deprecated](https://www.npmjs.com/package/opn).